KoansLib/KoanPrinter.php

```diff
@@ -20,7 +20,7 @@ public function printResult(TestResult $result): void
         $this->printFooter($result);
     }
 
-    protected function printHeader(): void
+    protected function printHeader(TestResult $result): void
```

Comment:
It's not being used, no?
```suggestion
    protected function printHeader(): void
```